.github/workflows/test-package.yml

```diff
@@ -43,7 +43,7 @@ jobs:
           sdk: ${{ matrix.sdk }}
       - id: install
         name: Install dependencies
-        run: dart pub get
```

Comment:
`dart test` is going to get you next!

Reply:
Haha, I shouldn't have bothered testing on `2.8.1`, I'm about to land the null safety migration and drop that anyway.